### DISCUSSION TITLE
maa无法启动

### DISCUSSION THREAD

**Discussion by jkj333** [MaaAssistantArknights/MaaAssistantArknights#5117] [General]
MAA 遇到了问题
未能加载文件或程序集“System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”或它的某一个依赖项。系统找不到指定的文件。
解决方案
出现未知错误
详细信息
System.IO.FileNotFoundException: 未能加载文件或程序集“System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”或它的某一个依赖项。系统找不到指定的文件。
文件名:“System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”
   在 System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
   在 System.Signature..ctor(IRuntimeFieldInfo fieldHandle, RuntimeType declaringType)
   在 System.Reflection.RtFieldInfo.get_FieldType()
   在 StyletIoC.Creation.BuilderUpper.<>c__DisplayClass6_0.<GetExpression>b__0(FieldInfo x)
   在 System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   在 System.Linq.Enumerable.<ConcatIterator>d__59`1.MoveNext()
   在 System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 StyletIoC.Creation.BuilderUpper.GetExpression(Expression inputParameterExpression, ParameterExpression registrationContext, Type type)
   在 StyletIoC.Internal.Creators.CreatorBase.CompleteExpressionFromCreator(Expression creator, ParameterExpression registrationContext)
   在 StyletIoC.Internal.Creators.TypeCreator.GetInstanceExpression(ParameterExpression registrationContext)
   在 StyletIoC.Internal.Registrations.RegistrationBase.GetGeneratorInternal()
   在 StyletIoC.Internal.Registrations.RegistrationBase.GetGenerator()
   在 StyletIoC.Internal.Container.Get(RuntimeTypeHandle typeHandle, String key, Type typeIfAvailable)
   在 StyletIoC.Internal.Container.Get[T](String key)
   在 MaaWpfGui.Helper.Instances.Instantiate(IContainer container)
   在 Stylet.BootstrapperBase.Start(String[] args)
   在 System.Windows.Application.OnStartup(StartupEventArgs e)
   在 System.Windows.Application.<.ctor>b__1_0(Object unused)
   在 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   在 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

警告: 程序集绑定日志记录被关闭。
要启用程序集绑定失败日志记录，请将注册表值 [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD)设置为 1。
注意: 会有一些与程序集绑定失败日志记录关联的性能损失。
要关闭此功能，请移除注册表值 [HKLM\Software\Microsoft\Fusion!EnableLog]。

**Comment by SherkeyXD** [comment]
遇到 bug 建议发 issue 并附上日志文件
